util/generate-rsa-signing-tests.py

```diff
@@ -26,10 +26,21 @@
 from cryptography.hazmat.primitives.asymmetric import rsa, padding
 import sys, copy
 
-def parse(fn):
+digest_output_lens = {
```

Comment:
I would rename this to `DIGEST_OUTPUT_LENGTHS`. The use of upper case is customary for constants in Python, and `lens` is hard to read as a multiple of `len`. Also, I would add the trailing comma for the last key-value pair.